js/server/modules/@arangodb/foxx/router/request.js

```diff
@@ -23,7 +23,10 @@
 // //////////////////////////////////////////////////////////////////////////////
 
 const parseUrl = require('url').parse;
-const {join: joinPath, resolve: resolvePath} = require('path').posix;
+const {
+  posix: {join: joinPath},
+  resolve: resolvePath
+} = require('path');
```

Comment:
Socket paths (i.e. file paths) are platform-dependent and need to be resolved locally. `joinPath` is only used on URLs and thus always needs to use POSIX.